### PR TITLE
[Task][Web2print]: Remove support for HeadlessChrome

### LINKED ISSUE #14512: [Task]: Remove support for HeadlessChrome
### Task description

Remove the support for HeadlessChrome
Check https://github.com/pimcore/pimcore/pull/14457

### PR DESCRIPTION
**Pull request by kingjia90**
Resolves https://github.com/pimcore/pimcore/issues/14512

### CODE REVIEW

**Review by mattamon — CHANGES_REQUESTED** [review]
Can you please resolve the conflicts :)

File: doc/03_Documents/02_Document_Types/15_Print_Documents.md [hunk 1]
```diff
@@ -44,8 +44,8 @@ Pimcore ships with default implementations (`Web2PrintController`, `containerAct
 
 Both web-to-print documents have an additional tab that is the place for rendering documents to print-ready PDFs.
 When rendering, the print is first rendererd to HTML, then rendered as a Twig template (yes, you can use Twig expressions in the document) and then rendered to an PDF. PDF rendering itself is done by an
-third party renderer. Currently we support [pdfreactor](http://www.pdfreactor.com/) 
-and [Headless Chrome](https://developers.google.com/web/tools/puppeteer) (deprecated). 
+third party renderer. Currently we support [pdfreactor](https://www.pdfreactor.com/) 
+and [Chromium](https://chromedevtools.github.io/devtools-protocol/).
```
Comment (mattamon) [inline doc/03_Documents/02_Document_Types/15_Print_Documents.md]: Isn't gotenberg missing here?

File: doc/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md [hunk 2]
```diff
@@ -43,9 +43,10 @@
 - [Config] `Pimcore\Config\Config` has been removed, see [#12477](https://github.com/pimcore/pimcore/issues/12477). Please use the returned array instead, e.g.
   ```php
   $web2printConfig = Config::getWeb2PrintConfig();
-  $web2printConfig = $web2printConfig['headlessChromeSettings'];
+  $web2printConfig = $web2printConfig['chromium'];
```
Comment (mattamon) [inline doc/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md]:
```suggestion
  $web2printConfig = $web2printConfig['chromiumSettings'];
```

**Review by mattamon — CHANGES_REQUESTED** [review]
Overall looks fine

File: doc/03_Documents/02_Document_Types/15_Print_Documents.md (hunk 1 above)
Comment (kingjia90) [inline doc/03_Documents/02_Document_Types/15_Print_Documents.md]: nice catch, adding a suggestion to commit on git
```suggestion
third party renderer. Currently we support [pdfreactor](https://www.pdfreactor.com/), [Chromium](https://chromedevtools.github.io/devtools-protocol/) and [Gotenberg](https://gotenberg.dev/). 
```

**Review by mattamon — APPROVED** [review]
LGTM

**Pull request merged**